go.mod

```diff
@@ -133,7 +133,7 @@ require (
 	google.golang.org/protobuf v1.28.1 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
-	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Is there a reason to have yaml.v2 and yaml.v3 both imported?

Reply:
Looks like it is already included in the [main branch](https://github.com/kubernetes-sigs/cluster-api/blob/main/go.mod#L136). After I changed `rollout_viewer.go` to use yaml.v3, it is still imported indirectly 🤔 .